src/lib/services/accountService.ts

```diff
@@ -18,12 +20,7 @@ export const useAccountBalances = (
       currentChainRecord?.name,
       currentChainRecord?.chain.chain_id,
     ],
-    async () =>
-      getAccountBalanceInfo(
-        walletAddress as Addr,
-        currentChainRecord?.name,
-        currentChainRecord?.chain.chain_id
-      ),
+    async () => getAccountBalanceInfo(baseApiRoute, walletAddress as Addr),
     { enabled: !!currentChainRecord || !!walletAddress }
```

Comment:
replace `currentChainRecord` by `balancesApiRoute`

Reply:
will just remove it since api route will never be falsy